### DISCUSSION TITLE
依赖引入老是失败

### DISCUSSION THREAD

**Discussion by HFYGitHub** [simple-robot/Discussions#16] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/45265631/198000886-82ebce50-b28c-4871-9e6f-6aa22c55b3ae.png)
尝试降级版本也不行，换了maven@EnabledSimBot也会爆红，是我网络原因吗

**Comment by ForteScarlet** [comment, marked as answer]
> Could not resolve net.mamoe:mirai-core:2.12.1

这似乎是mirai的依赖下载失败导致的，你可以尝试清理本地maven仓库的 `*.lastUpdated` 文件、刷新等操作尝试重新导入（如果配置了什么镜像则试着不用镜像之类的）

> @EnabledSimBot也会爆红

`@EnableSimbot` 爆红的问题可以先参考下 [这里](https://www.yuque.com/simpler-robot/simpler-robot-doc/rvsl9p)

**Reply by HFYGitHub** [reply]
谢谢解答，原因是我配置了阿里云镜像，删掉就可以了
